### ISSUE TITLE
Parsing error with nested zero-argument functions

### ISSUE THREAD

**Issue by EvanBalster** [EvanBalster/TiddlyWikiFormula#8]
This formula causes an unexpected result, where the SWITCH statement is treated as if it terminates at the right parenthesis in `now()`:

```
switch(weekday(now()),
  1, "Sun", 2, "Mon", 3, "Tue", 4, "Wed",
  5, "Thu", 6, "Fri", 7, "Sat",
  "ERROR")
```

Removing the parentheses after `now` causes the problem to go away (because parentheses are currently optional for zero-argument functions).

**Comment by EvanBalster** [comment]
Fixed in https://github.com/EvanBalster/TiddlyWikiFormula/commit/95f172bc0bac91e546bb3f9f5b29eb0a141e9102

**Comment by EvanBalster** [comment]
Released fix in 0.1.3.
